### PR TITLE
Documentation

### LINKED ISSUE #13: Create readthedocs/sphinx site
The README is becoming large and mixes user and developer docs. 

It would be nice to have a documentation site using Sphinx and hosted on Readthedocs.

Use https://i-vresse-bartender.readthedocs.io/ as https://bartender.readthedocs.io/ is already taken.

README could be split into
* CONTRIBUTING.md
* docs/index.md
* docs/usage.md
* docs/user_management.md
* docs/configuration.md

### LINKED ISSUE #27: Add Code of conduct and contributing guidelines

### LINKED ISSUE #28: Add markdown linter to CI and/or pre-commit
The Markdown files should be formatted using a style.

Use https://github.com/markdownlint/markdownlint to enforce style.

### PR DESCRIPTION
**Pull request by Peter9192**
Docs preview: https://i-vresse-bartender--49.org.readthedocs.build/en/49/index.html#

Split README.md into multiple separate pages in the documentation.

To do:

- [x] Split up README.md
- [x] Add CONTRIBUTING.md
- [x] Add code of conduct
- [x] Use myst-parser
  - [x] Development version
  - [ ] Use released version of myst-parser with sphinx v6 compatibility
- [x] Add markdown linter
- [x] Enable ReadTheDocs integration
- [x] Rewrite (parts of) documentation (to be more in line with the new structure)

closes #13 
closes #28 
closes #27

### CODE REVIEW

**Comment by Peter9192** [comment]
test:

1. one
1. two

**Comment by Peter9192** [comment]
see https://github.com/executablebooks/MyST-Parser/pull/664 for status of myst-parser release

File: docs/develop.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
+# Detailed instructions
```
Comment (sverhoeven) [inline docs/develop.md:1]: Seems a bit odd tittle.
```suggestion
# Developer guidelines
```

**Review by sverhoeven — APPROVED** [review]
Looks much better.

Can you invite me on rtd use my work email?

Just a small rename

**Pull request merged**